scripts/check_doc_ipfs_hashes.py

```diff
@@ -54,6 +54,7 @@
 HASH_SKIPS = (
     "bafybeie553shfmnds6v7defynjv5kmjkqf2aygj345jbbcssevtnkbodbe",
     "bafybeie6zryhy7v3fb4mwlu3br4brfxva4m5j2fyldbclq2v3d6modmewm",
+    "bafybeieo2gmyuut6chwnonutmcxo75wz7mpjxtim6c2naaarqpp5wa46ge",
```

Comment:
why

Reply:
Because the price oracle service does not belong to this repository and the IPFS hash check script will not find it in packages.json.